doc/classes/ProjectSettings.xml

```diff
@@ -2305,6 +2305,9 @@
 		<member name="rendering/rendering_device/driver.windows" type="String" setter="" getter="" default="&quot;vulkan&quot;">
 			Windows override for [member rendering/rendering_device/driver].
 		</member>
+		<member name="rendering/rendering_device/pipeline_cache/save_interval_mb" type="float" setter="" getter="" default="3.0">
```

Comment:
This wording makes it sound as if this was about time instead of size. Let me suggest `save_increment_mb`, for instance.

Reply:
One of the meanings of interval is a gap between two points. The setting describes a gap between last time we saved and next time. It could be maybe ```save_gap_mb``` or ```save_delta_mb``` if not ```interval```?

Reply:
Could be `save_chunk_size_mb`?